Classes/GraphQL/Resolver/Type/AssetResolver.php

```diff
@@ -109,7 +109,7 @@ public function file(Types\Asset $asset): ?Types\File
 
         $icon = $this->fileTypeIconService::getIcon($assetProxy->getFilename());
 
-        if ($asset instanceof ProvidesOriginalUriInterface) {
+        if ($assetProxy instanceof ProvidesOriginalUriInterface) {
             /** @phpstan-ignore method.notFound */
             $url = (string)$assetProxy->getOriginalUri();
```

Comment:
```suggestion
        if ($asset instanceof ProvidesOriginalUriInterface) {
            /** @phpstan-ignore method.notFound */
            $url = (string)$asset->getOriginalUri();
```

Reply:
I've tested it and the original url from the asset procy doesn't work. But: If I change it to the code above, it worked. Also with images, videos, etc